### PR TITLE
Console - Add favorite on roles

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: console/src/main/webapp/manager/app/app.es6 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class AppController {
-  static $inject = [ '$scope', '$router', '$location' ]
+  static $inject = [ '$scope', '$router', '$location' , 'roleAdminList']
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/app.es6]: extra space before comma

Comment (fvanderbiest) [reply console/src/main/webapp/manager/app/app.es6]: missing space before ]

File: console/src/main/webapp/manager/app/app.es6 [hunk 2]
```diff
@@ -30,6 +30,7 @@ class AppController {
     $scope.isActive = (routes) => routes.some(
       route => $location.$$path.indexOf(route) === 1
     )
+    $scope.isProtected = role => roleAdminList().indexOf(role.cn) !== -1
```
Comment (fvanderbiest) [inline console/src/main/webapp/manager/app/app.es6]: $scope.isProtectedRole would be more self-explicit

File: console/src/main/webapp/manager/app/app.es6 (hunk 1 above)
Comment (tonio) [reply console/src/main/webapp/manager/app/app.es6]: `npm run lint` should catch all of these for you

**Pull request merged**